TranslateKit/Translation.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Translation.swift
+//  TranslateKit
+//
+//  Created by Jennifer on 13/02/2016.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import UIKit
+
+public struct Translation {
+    
+    public let meanings: [Meaning]
+    public let additionalMeanings: [Meaning]
+    public let compoundMeanings: [Meaning]
+    
+    public init?(dictionary:JSONDictionary) {
+        guard let termDictionaries = dictionary["term0"] as? JSONDictionary,
+            meaningDictionaries = termDictionaries["PrincipalTranslations"] as? JSONDictionary else { return nil }
+        let sortedDictionaries = meaningDictionaries.sort({ $0.0 < $1.0 })
+        self.meanings = sortedDictionaries.flatMap {
+            guard let dictionary = $0.1 as? JSONDictionary else { return nil }
+            return Meaning(dictionary: dictionary)
+        }
```

Comment:
What about creating a method to do that since we do the exact same thing for `additionalMeaningDictionaries` and `compoundMeaningDictionaries`?